salt/states/zabbix_host.py

```diff
@@ -125,6 +125,20 @@ def _interface_format(interfaces_data):
 
     interfaces_formated = _interface_format(interfaces)
 
+    # Ensure groups are all groupid
+    groupids = []
+    for group in groups:
+        if isinstance(group, basestring):
```

Comment:
We need this to be `if isinstance(group, six.string_types):` and a `from salt.ext import six` up top, please.